### PR TITLE
ci: add current node version

### PR DESCRIPTION
**Pull request by tschmidtb51**

### CODE REVIEW

**Comment by tschmidtb51** [comment]
@jasoniangreen Could you please have a look?

**Comment by jasoniangreen** [comment]
Hi @tschmidtb51, thanks for raising this. Let's just do the same as `ajv` main repo and have 16, 18, 20 and 21.

I wonder why none of the builds are actually completing? I will look into it soon.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 12.x, 14.x, 16.x, 18.x, 19.x, 20.x]
```
Comment (jasoniangreen) [inline .github/workflows/build.yml:15]:
```suggestion
        node-version: [16.x, 18.x, 20.x, 21.x]
```

**Comment by tschmidtb51** [comment]
> I wonder why none of the builds are actually completing? I will look into it soon.

I think the issue is that 10, 12 and 14 where removed but are still required for a merge into the master branch...

**Comment by jasoniangreen** [comment]
> > I wonder why none of the builds are actually completing? I will look into it soon.
> 
> I think the issue is that 10, 12 and 14 where removed but are still required for a merge into the master branch...

Yeah, and I don't have permission to change that. I have reached out to the repo owner.

**Review by jasoniangreen — APPROVED** [review]

**Pull request merged**